### DISCUSSION TITLE
Formula- issue with Concat & space in column name [Jul 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2805] [Discord Threads]
> 🙋‍♂️ **edon**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@306585454507982849> please help us with your project configurations (open project, right top menu, copy project info).  
#  
> 👨‍🦲 **dstala**  
#### I tried with following project info, worked well   
####    
#### Node: **v16.14.0**   
#### Arch: **arm64**   
#### Platform: **darwin**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.92.3**  
#  
> 👨‍🦲 **dstala**  
#### & do specify the type of the field "Item Name"  
#  
> 🙋‍♂️ **ideeb**  
#### Copy project info only gives me: keyQBkWiM6kQde8Wp  
#  
> 🙋‍♂️ **ideeb**  
#### I've tried it with, text, numbers, and lookups  with no success  
#  
> 👨‍🦲 **dstala**  
#### strange. what version of nocodb are you on? you can hover around nocodb logo to find it out.  
#  
> 🙋‍♂️ **ideeb**  
#### 0.92.3  
#  
> 🙋‍♂️ **ideeb**  
#### Node: **v16.15.0**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **false**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **false**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.92.3**  
#  
> 🙋‍♂️ **ideeb**  
#### Both issues stemmed from using safari -- 🙊 no problems on chrome  
#  
> 👨‍🦲 **dstala**  
#### > Both issues stemmed from using safari -- 🙊 no problems on chrome   
#### ah, thats interesting.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/996209889095458907)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
